tools/kmodules/lisa-in-tree/fetch_lisa_module.py

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env python3
+
+#
+# Usage:
```

Comment:
This doc comment should be cleaned up to remove CLI option doc (that's autogenerated by argparse --help) and the rest should go in  `ArgumentParser(description=...)` so it is shown in `--help`